content/developer/reference/user_interface/view_architectures.rst

```diff
@@ -2520,53 +2519,287 @@ Optional attributes can be added to the root element `kanban` to customize the v
+               </t>
+            </templates>
+         </kanban>
+
+.. attribute:: widget
+   :noindex:
+
+   An object with 2 keys defining the available actions for the user:
+
+   - `editable`: true if the user can edit records, false otherwise;
+   - `deletable`: true if the user can delete records, false otherwise.
+
+   This is useful to conditionally display elements requiring specific access rights.
+
+   .. example::
+      .. code-block:: xml
+
+         <kanban>
+            <templates>
+               <t t-name="card">
+                  <field name="name"/>
+               </t>
+               <t t-name="menu">
+                  <a t-if="widget.deletable" role="menuitem" type="delete" class="dropdown-item">Delete</a>
+               </t>
+            </templates>
+         </kanban>
+
+.. attribute:: context
+   :noindex:
+
+   The current context propagated from either the action that opens the kanban view, or the one2many
+   or many2many field that embeds the kanban view in a form view.
+
+.. attribute:: read_only_mode
+   :noindex:
+
+   Indicates that the view is readonly.
+
+   :type: bool
+
+.. attribute:: selection_mode
+   :noindex:
+
+   Whether the kanban view is opened when selecting a many2one or many2many field (in mobile
+   environment).
+
+   :type: bool
+
+.. attribute:: luxon
+   :noindex:
+
+   The `luxon <https://moment.github.io/luxon/api-docs/index.html>`_ object, allowing to
+   manipulate date and datetime field values.
+
+.. attribute:: JSON
+   :noindex:
+
+   The Javascript `JSON <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON>`_
+   namespace object containing a `parse` method allowing to parse json field values into Javascript
+   Objects.
+
+Buttons and links
+*****************
+
+While most of the kanban templates are standard :ref:`QWeb templates <reference/qweb>`, the kanban
+view processes `button` and `a` elements is a special way. Buttons and links with a `type` attribute
+perform different operations than their standard HTML function. The `type` attribute can have the
+values `action` and `object` of :ref:`regular buttons <reference/view_architectures/list/button>`,
+or the following values:
+
+  .. attribute:: open
+     :noindex:
+
+     Clicking the element opens the card's record in form view.
+
+  .. attribute:: delete
+     :noindex:
+
+     Clicking the element deletes the card's record and removes the card.
+
+  .. attribute:: archive
+     :noindex:
+
+     Clicking the element archives the card's record and removes the card.
+
+  .. attribute:: unarchive
+     :noindex:
+
+     Clicking the element unarchives the card's record and removes the card.
+
+  .. attribute:: set_cover
+     :noindex:
+
+     Clicking the element allows to select an image to set as cover image of the record.
+
+Widgets
+*******
+
+The `widget` element allows to insert dynamicaly generated (in Javascript) html inside the cards. It
```

Comment:
i would mention somewhere, maybe as a note, that widget are actually implemented as owl component